src/atlannot/merge/common.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2021, Blue Brain Project, EPFL
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Functionality common to all atlas merge strategies."""
+from __future__ import annotations
+
+import numpy as np
+
+from atlannot.atlas.region_meta import RegionMeta
+
+
+def replace(array: np.ndarray, old_value: int, new_value: int) -> None:
+    """Replace integer values in a numpy array.
+
+    Parameters
+    ----------
+    array
+        An arbitrary numpy array.
+    old_value
+        The value to replace.
+    new_value
+        The new value that replaces the old one.
+    """
+    array[array == old_value] = new_value
+
+
+def atlas_remap(
```

Comment:
Just thinking about this now, maybe we could replace the remapping of labels with that function instead ? I know we use it in `ants.transform` and it is a bit slow. What do you think ? Should we create a ticket for this ?

Reply:
Yes, very good point! Maybe we can do that in a separate issue?

Reply:
Sounds a good plan :)

Reply:
Created #23